documentation/largescaledeployment.md

```diff
@@ -3,7 +3,18 @@ title: Maximum scale supported by VCP
 ---
 **Maximum scale validated - 1000 nodes Kubernetes cluster**
 
-The following configuration is used to validate the scale limits for VCP.
+**Test Scenario**
+1. Creating a kubernetes cluster with 1000 worker nodes and enabling VCP on all of them
```

Comment:
expand VCP here and use the abbreviation in other places. Reader would not know what VCP means the first time.